build.gradle

```diff
@@ -1,29 +1,28 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 buildscript {
     repositories {
-        jcenter()
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
         google()
+        jcenter()
     }
+
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.2'
+        classpath 'com.android.tools.build:gradle:3.6.1'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
     }
 }
 
+plugins {
+    id 'lifecycle-base'
```

Comment:
Something to add to https://github.com/auth0/oss-library-gradle-plugin/?

Reply:
No, since this a root project thing. Unless you want to create a new plugin to configure the root project?

Reply:
What do you mean by root project? Maybe I'm missing something, but if I'm correct, you replaced the "clean" task we had with this lifecycle plugin, which seems to remove the build folder for us. You are just extending that to remove the temporary changelog file, which this same oss library plugin we had generates. 
Wouldn't it make sense for this oss library plugin to apply internally the lifecycle-base to both java and android libs, and also extend it to remove the temporary changelog?